ansible/roles/software/common/php-fixes/defaults/main.yml

```diff
@@ -4,7 +4,8 @@ php_cli_max_execution_time: 600
 php_cli_memory_limit: 2048M
 
 # Name of the PHP FPM service
-php_fpm_service_name: php{{ php_major_minor_version }}-fpm
+php_fpm_service_name: "php{{ php_major_minor_version }}-fpm"
 
 # Major/minor version of PHP
-php_major_minor_version: "7.0"
+php_major_minor_version: "{{ php_version }}"
+php_version: 7.3
```

Comment:
FWIW, I like the `php_major_minor_version name`, because it's more descriptive. Technically, the PHP version would include a patch number (7.3.x) and I think the `php_version` was a poor choice on geerlingguy. And it's only now that I realize we could've reversed these variables to accomodate the third-party roles:

```
php_major_minor_version: "7.3"
php_version: "{{ php_major_minor_version }}"
```

Not a deal breaker though.

Reply:
I thought about this too late while working on the branch... I agree, and made the change in another push.